goentri.com.owner.json

```diff
@@ -0,0 +1,27 @@
+{
+  "providerId": "goentri.com",
+  "providerName": "Entri",
+  "serviceId": "owner",
+  "serviceName": "Owner",
+  "version": 1,
+  "logoUrl": "https://cdn.goentri.com/logo.svg",
+  "description": "Allows user to point domain to the website",
+  "syncPubKeyDomain": "goentri.com",
+  "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
+  "variableDescription": "a is  IP record",
+  "sharedProviderName": true,
+  "records": [
+    {
+      "type": "A",
+      "host": "@",
+      "pointsTo": "%a%",
```

Comment:
As suggested by wdbaker-godaddy,  can we please consider using variable %ip% here? We believe some DNS providers have trouble with single character variables. Thanks!

Reply:
Thanks for the suggestion @vkasam-godaddy 
I updated variable name.